src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/StructureFormMetadataLoader.php

```diff
@@ -86,6 +86,8 @@ public function getMetadata(string $key, string $locale, array $metadataOptions
         if (isset($metadataOptions['webspace'])) {
             $webspace = $this->webspaceManager->findWebspaceByKey($metadataOptions['webspace']);
 
+            $form->setDefaultType($webspace->getDefaultTemplate($key));
```

Comment:
Missing that for example the default type is correctly set by the `sulu.content.structure.default_types` parameter which is used for snippets and articles

```php
$form->setDefaultType($this->defaultTypes[$key] ?? null);
```

```yaml
sulu_core:
    content:
        structure:
            default_type:
                snippet: default
```